Client/src/components/common/Table/index.css

```diff
@@ -0,0 +1,33 @@
+.Table {
+  margin: 50px auto;
```

Comment:
remove ``` margin : 50px  auto; ```
you don't need to determine the place of the table in the page (illegal thing) the user of the table will determine where that should appear.
that will make a lot of problems with other teammates